examples/live/run.sh

```diff
@@ -195,13 +195,27 @@ python -m mpi4py `which pycbc_live` \
 --src-class-eff-to-lum-distance 0.74899 \
 --src-class-lum-distance-to-delta -0.51557 -0.32195 \
 --run-snr-optimization \
+--optimizer differential_evolution \
+--differential-evolution-maxiter 50 \
+--differential-evolution-popsize 100 \
+--include-candidate-in-optimizer \
+--optimizer-seed 42 \
```

Comment:
Can we make it a bit clearer that those options are related to the SNR optimization? Maybe prefixing everything with `--snr-opt`?

Reply:
I would worry that argparse doesn't cope well with very long option names so we might have to prefer eg `--snr-opt-di-maxiter`

Reply:
Some examples which do not seem that much longer to me:
```
--snr-opt-method differential_evolution
--snr-opt-di-max-iter
--snr-opt-di-pop-size
--snr-opt-include-candidate
--snr-opt-seed
```

Reply:
I have updated the argument names to all include the prefix 'snr-opt'. This requires a bit of code changes because the argument given to 'snr-opt-method' is 'differential-evolution' but the options pertaining to that method are all labelled 'snr-opt-di-*' to keep them shortened.